online_store/gt_sports.html

```diff
@@ -0,0 +1,100 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
```

Comment:
Indentation: 
Ex.:
```javascript
...
<body>
    <h2 class="column" id="h2title">goodTeens - Sports</h2>
    <section class="card" id="product1"> 
        <h3 class="productName1"></h3>
    </section>
    ...
</body>
```
If you structure your code properly then it's easier for you to read it, easier to maintain and debug if you have some errors.